__tests__/endpoint.test.js

```diff
@@ -112,3 +112,23 @@ describe("GET: /api/articles", () => {
       });
   });
 });
+
+// Test Task 5: --------------------
+describe("GET: /api/articles/:article_id", () => {
+  test("200: Returns articles object with required properties", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+      });
+  });
+});
```

Comment:
I'd add two tests here for error handling.
What if the Article ID doesn't exist?
What if something other than a number is passed in as an article ID